websites/docs/astro.config.mjs

```diff
@@ -38,16 +38,30 @@ export default defineConfig({
             {
               label: `Glossary`,
               link: `/en/getting_started/glossary`,
+            }
+          ],
+        },
+        {
+          label: `Deployment Guides`,
+          collapsed: true,
+          items: [
+            {
+              label: `Local`,
+              link: `/en/getting_started/local_development`,
             },
             {
-              label: `Installation`,
-              link: `/en/getting_started/installation`,
+              label: `Docker`,
+              link: `/en/getting_started/local_deployment`,
             },
             {
-              label: `Deployment`,
-              link: `/en/getting_started/deployment`,
+              label: `Ansible `,
+              link: `/en/getting_started/ansible_deployment`,
             },
-          ],
+            {
+              label: `Kubernetes Deployment`,
+              link: `/en/getting_started/kubernetes_deployment`,
+            }
+          ]
```

Comment:
Let's move the guides to the following;

- Learn
   - Deployment

Reply:
Addressed